src/frameworks/index.ts

```
@@ -513,6 +514,12 @@ ${firebaseDefaults ? `__FIREBASE_DEFAULTS__=${JSON.stringify(firebaseDefaults)}\
         // continue
       });
 
+      if (fileExistsSync(getProjectPath(".npmrc"))) {
+        await copyFile(getProjectPath(".npmrc"), join(functionsDist, ".npmrc")).catch(() => {
```

I think we can drop the catch and let the error throw

---

Otherwise looks good. Mind adding a changelog entry?

---

No worries, done!